exporter/datadogexporter/translate_traces_test.go

```diff
@@ -577,6 +634,32 @@ func TestSpanNameTranslation(t *testing.T) {
 	assert.Equal(t, strings.ToLower(fmt.Sprintf("%s.%s", "hyphenated_value", strings.TrimPrefix(pdata.SpanKindSERVER.String(), "SPAN_KIND_"))), spanNameHyphen)
 }
 
+// ensure that the datadog span name uses IL name +kind when available and falls back to opetelemetry + kind
+func TestSpanNameNormalization(t *testing.T) {
+	emptyName := ""
+	dashName := "k-e-b-a-b"
+	camelCaseName := "camelCase"
+	periodName := "first.second"
+	removeSpacesName := "removes Spaces"
+	numericsName := "num3r1cs_OK"
+	underscoresName := "permits_underscores"
+	tabName := "\t"
+	junkName := "\tgetsRidOf\x1c\x1c\x18Junk"
+	onlyJunkName := "\x02\x1c\x18\x08_only_junk_"
+
+	assert.Equal(t, utils.NormalizeSpanName(emptyName, false), "")
+	assert.Equal(t, utils.NormalizeSpanName(dashName, false), "k_e_b_a_b")
+	assert.Equal(t, utils.NormalizeServiceName(dashName), "k-e-b-a-b")
+	assert.Equal(t, utils.NormalizeSpanName(camelCaseName, false), "camelcase")
+	assert.Equal(t, utils.NormalizeSpanName(periodName, false), "first.second")
+	assert.Equal(t, utils.NormalizeSpanName(removeSpacesName, false), "removes_spaces")
+	assert.Equal(t, utils.NormalizeSpanName(numericsName, false), "num3r1cs_ok")
+	assert.Equal(t, utils.NormalizeSpanName(underscoresName, false), "permits_underscores")
+	assert.Equal(t, utils.NormalizeSpanName(tabName, false), "")
+	assert.Equal(t, utils.NormalizeSpanName(junkName, false), "getsridof_junk")
+	assert.Equal(t, utils.NormalizeSpanName(onlyJunkName, false), "only_junk")
+}
```

Comment:
Let's add a couple more tests to increase coverage, something like:

```suggestion
 	assert.Equal(t, utils.NormalizeServiceName("\x02\x1c\x18\x08"), DefaultServiceName)
 	assert.Equal(t, utils.NormalizeServiceName(""), DefaultServiceName)
}
```

Otherwise LGTM

Reply:
👍 updated